packages/portal/swagger-ui-web-component/package.json

```diff
@@ -16,8 +16,8 @@
     "@kong/swagger-ui-kong-theme-universal": "^4.3.3",
     "react": "18.3.1",
     "react-dom": "18.3.1",
-    "swagger-client": "^3.29.3",
-    "swagger-ui": "^5.17.14"
+    "swagger-client": "3.26.7",
+    "swagger-ui": "5.7.2"
```

Comment:
These will likely still trigger a Renovate PR in a future version bump. We can either choose to add a config setting to this repository in `/renovate.json` to ignore these packages, or, more desirable, someone can update the styles for the newer swagger versions.